sites/skeleton.dev/src/lib/themes.ts

```diff
@@ -1,7 +1,22 @@
 // Preset Themes
 
-const ghPath = 'https://github.com/skeletonlabs/skeleton/blob/master/src/themes';
-export const themes: any[] = [
+interface Font {
```

Comment:
We are missing a `url:string` here, the url is used as href for the buttons in the theme.css.
we should also provide a url for each font in `themes`.
this solves another 2 of the svelte-check errors.

Reply:
Still waiting on more explanation.

Reply:
Tentative change in place.